packages/engine/bin/cli/src/exsrv.rs

```diff
@@ -158,7 +160,7 @@ impl Server {
         };
         result_tx
             .send(res)
-            .map_err(|_| format_err!("Sending server control result"))?;
+            .map_err(|_| report!("Sending server control result"))?;
```

Comment:
Wonder if this would be a good opportunity to revisit all `map_err(|_| ..`s and see if they should have a `wrap_err_lazy` or similar instead. (Like you did below, if this one was consciously kept then all good)

Reply:
The return type of the `Err` arm of `send(res)` does not implement `std::error::Error`, thus `wrap_err` is not available. `map_err` is from `std::result::Result`, not from `ResultExt` or similar.